clinica/pipelines/t1_freesurfer/t1_freesurfer_utils.py

```diff
@@ -89,33 +101,33 @@ def save_to_caps(source_dir, image_id, caps_dir, overwrite_caps=False):
         'freesurfer_cross_sectional'
     )
 
-    log_file = os.path.join(os.path.expanduser(source_dir), image_id, image_id, 'scripts', 'recon-all.log')
-    if os.path.isfile(log_file):
+    representative_file = os.path.join(image_id, 'mri', 'aparc+aseg.mgz')
```

Comment:
This is one of the final files generated by FreeSurfer.

Reply:
Why having stopped to look into the recon-all.log file for freesurfer status ?

Reply:
I explained it in the documentation of the function:
> We do not need to check the line "finished without error" in scripts/recon-all.log.
If an error occurs, it will be detected by Nipype and the next nodes (including
save_to_caps will not be called).

So, I enforced the presence of one on the latest file in the output folder.